content/rs/references/rest-api/objects/state-machine.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: State Machine object
+linkTitle: state-machine
+description: An object that represents state machines.
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+
+A `state-machine` object tracks the creation of a database on a Redis Enterprise cluster. When you create a database on a Redis Enterprise cluster, Redis Enterprise creates a state machine that tracks the status of the database creation process.
```

Comment:
Can you streamline this paragraph?

Reply:
Fixed in https://github.com/RedisLabs/redislabs-docs/pull/1885/commits/e116a51a5dba34fbce7b05b65a861081441f7257.